assets/constants.js

```diff
@@ -19,6 +19,8 @@ var constants = {
     "Mathematics", "Medicine", "Music", "Philosophy", "Physics", "Poetry",
     "Probability", "Programming", "Puzzles", "Reading", "Spanish", "Sport",
     "Statistics", "Trigonometry", "Welcome"],
+  "ACTIVITY_TYPE_EXPLORATION": "exploration",
```

Comment:
Nope, we should have a single source of truth. It's fine to use `constants` throughout the backend, too.